ability_test.go

```diff
@@ -12,14 +12,13 @@ func TestAbility_ListAbilities(t *testing.T) {
 
 	mux.HandleFunc("/abilities", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "GET")
-		w.Write([]byte(`{"abilities": ["sso"]}`))
+		_, _ = w.Write([]byte(`{"abilities": ["sso"]}`))
 	})
 
-	var client = &Client{apiEndpoint: server.URL, authToken: "foo", HTTPClient: defaultHTTPClient}
+	client := &Client{apiEndpoint: server.URL, authToken: "foo", HTTPClient: defaultHTTPClient}
```

Comment:
My editor uses `gofumpt` instead of `go fmt` for style*, and it's a bit more opinionated. This was changed automatically.

* Technically it's `gofumports`, which is meant to replace `goimports`.